src/main/java/com/sphy/hotelmanagementapplication/domain/Admin.java

```diff
@@ -19,7 +19,10 @@ public Admin(Long id, boolean emailVerify, String username, String firstname, St
         super(id, emailVerify, username, firstname, lastname, email, Role.ADMIN);
     }
 
-	public Long getId() {
+    public Admin(long l) {
```

Comment:
use super.setId() to set the id for the newly created object that used this constructor